src/rules.jl

```diff
@@ -17,14 +17,14 @@ Examples:
 
 unary input, unary output scalar function:
 
-```jldoctest
-julia> dself = Zero()
-Zero()
+```jldoctest frule
+julia> dself = NO_FIELDS;
 
-julia> x = rand();
+julia> x = rand()
+0.8236475079774124
 
-julia> sinx, Δsinx = frule(sin, x, dself, 1)
-(0.35696518021277485, 0.9341176907197836)
+julia> sinx, Δsinx = frule((dself, 1), sin, x)
+(0.7336293678134624, 0.6795498147167869)
```

Comment:
This is because the value of `x` changed, right?

Reply:
yep